static/templates/subscription_settings.handlebars

```diff
@@ -28,7 +28,12 @@
   </div>
   <div class="regular_subscription_settings collapse {{#subscribed}}in{{/subscribed}}">
     <div class="subscription-type">
-      {{partial "subscription_type"}}
+      <div class="subscription-type-text">
+        {{partial "subscription_type"}}
+      </div>
+      {{#if is_admin}}
+        <a class="change-stream-privacy" data-stream-id="{{stream_id}}" data-is-private="{{#if can_make_public}}true{{else}}false{{/if}}">{{t "[Change]"}}</a>
```

Comment:
I don't think you need the `data-stream-id` here; it's accessed via the `.subscription_settings` container object anyway, right?  That's why none of these other things have duplicate `data-stream-id` declarations...